source/slang/slang-emit-metal.cpp

```diff
@@ -494,7 +494,7 @@ void MetalSourceEmitter::emitSimpleTypeImpl(IRType* type)
         case kIROp_ParameterBlockType:
```

Comment:
I guess the answer is yes but is it correct to give "constant" keyword for parameter block type?

Reply:
IIRC, according to the metal spec, constant memory space is what maps to a ConstantBuffer in d3d, or a uniform buffer in Vulkan.